prettier.config.js

```diff
@@ -0,0 +1,3 @@
+import prettierConfig from '@kouts/eslint-config/prettier'
+
+export default { ...prettierConfig, semi: true }
```

Comment:
I've set this to `semi: true` here as that was how it was set before but I've recently switched to not using `semi`s and I think it's actually better. 
This is the article that changed my mind: https://www.epicweb.dev/your-code-style-does-matter-actually

Reply:
I'm used to working without semicolons in Go, but I think they are more common and expected in JS/TS projects.

Reply:
Yep agree that it's more common to use them than not, that's why I've set `semi: true` 😄